widgetastic_manageiq/__init__.py

```diff
@@ -865,21 +871,43 @@ def __init__(self, parent, title, *args, **kwargs):
     @property
     def fields(self):
         """Returns a list of the field names in the table (the left column)."""
-        return [row[0].text for row in self]
+        fields_names = []
+        for index, field in enumerate(self):
+            if self.browser.get_attribute('class', field[0]):
```

Comment:
what is this for ?

Reply:
filtering rowspan fields, as don't have class attribute,  and in case we want to get field name for such field it returns category name, and later causes an error